.gitignore

```diff
@@ -1 +1,138 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
```

Comment:
You can try to change it to `*/**/__pycache__/` looks like still those cache files are being comitted.